Makefile

```diff
@@ -13,12 +13,12 @@ all clean:
 	done
 
 distclean:
-	rm mlt-config packages.dat; \
+	rm -f mlt-config packages.dat
 	list='$(SUBDIRS)'; \
 	for subdir in $$list; do \
 		$(MAKE) -C $$subdir $@ || exit 1; \
-	done; \
-	rm config.mak;
+	done
+	echo > config.mak
```

Comment:
config.mak is inconditionnaly included in all modules makefiles, so "make distclean" fails on this include if run on an already clean tree... but maybe this should be done before the loop!
an other solution would have been to surround all these includes by file existence checks...